Description/Enhancer/Sylius/ResourceEnhancer.php

```diff
@@ -81,14 +82,38 @@ public function enhance(Description $description)
         ];
 
         foreach ($map as $descriptor => $action) {
+            try {
+                $url = $this->urlGenerator->generate(
+                    $configuration->getRouteName($action),
+                    [
+                        'id' => $payload->getId(),
+                    ]
+                );
+                $description->set($descriptor, $url);
+            } catch (RouteNotFoundException $e) {
```

Comment:
Even if we did a "has route" the result would be the same: the descriptor would not be added. So I am OK with this solution.